src/AstClause.h

```diff
@@ -177,17 +177,7 @@ class AstClause : public AstNode {
 
     /** Add a Literal to the body of the clause */
     void addToBody(std::unique_ptr<AstLiteral> l) {
```

Comment:
While you're here, a more descriptive variable name would be nice. 'literal', for example.